panther_docking/test/test_panther_charging_dock.cpp

```diff
@@ -361,7 +259,7 @@ TEST_F(TestPantherChargingDock, UpdateDockPoseAndStagingPosePublish)
   SetDetectionOffsets();
   SetStagingOffsets();
 
-  ConfigureAndActivateDock(1.21);
+  ConfigureAndActivateDock();
   SetDockToBaseLinkTransform();
   SetBaseLinkToOdomTransform();
 
```

Comment:
How about creating a function for checking if two poses are "near"? Then you don't need to check it manually